library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/RegistryLibInitializer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.registry.impl;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.loader.api.ModContainer;
+import org.quiltmc.qsl.base.api.entrypoint.client.ClientModInitializer;
+import org.quiltmc.qsl.registry.impl.sync.ClientRegistrySync;
+
+@Environment(EnvType.CLIENT)
+public class RegistryLibInitializer implements ClientModInitializer {
```

Comment:
```suggestion
public class RegistrySyncClientInitializer implements ClientModInitializer {
```
Dont forget the file name too